config/routes.rb

```diff
@@ -537,6 +537,7 @@
   resource :redirect, controller: "redirect", only: [:show] do
     member do
       get :do_redirect
+      get :show
```

Comment:
You don't actually need this line, since the route is already defined in the "only: [:show]" bit above.